src/Aquarius.ONE.ClientSDK/Operations/Sample/SampleApi.cs

```diff
@@ -69,7 +72,245 @@ public async Task<bool> UpdateActivitiesAsync(List<Activity> activities)
             }
         }
 
+        public async Task<bool> CreateAnalyteAsync(Analyte analyte)
+        {
+            var watch = System.Diagnostics.Stopwatch.StartNew();
+
+            var requestId = Guid.NewGuid();
+            var endpoint = $"/operations/sample/v1/analyte";
+
+            var json = JsonConvert.SerializeObject(analyte, new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore });

```

Comment:
Could create a class instance of the JsonSerializerSettings and use it throughout the class when it's needed. Should also add the CamelCaseResolver.